### ISSUE TITLE
Jumping to a target, undoing, and trying to jump to any target with the same letter jumps to the first option

### ISSUE THREAD

**Issue by vimeitor** [yangmillstheory/vim-snipe#24]
### Mappings

omap l <Plug>(snipe-t)
omap L <Plug>(snipe-f)
omap h <Plug>(snipe-T)
omap H <Plug>(snipe-F)

### Steps to reproduce
"this is an example sentence"
If we try to jump to any target with the letter e (with snipe-t, for example) we get
"this is an example s**a**nt**s**nc**d**"
Jump to any character. Undo. Try to jump to any target with the letter e. You get none, and it automatically leaves:
"sentence"
since it's what you would get if it jumped to the first target

**Comment by globalchubby** [comment]
Sorry can you provide the vim commands to reproduce? The great thing about Vim is that we can be precise with what we're doing.

Also, are you missing some mappings? I only see mappings for operator-pending mode, these functions work in Normal mode and Visual mode as well.

**Comment by vimeitor** [comment]
Thank you for the answer. I actually only have omaps, since that's what I use the most.

For the sentence, I just press **dle**, select target (for example, f), then undo, then press **dle** again (so the whole thing would be **dlefudle**. Here's how it looks:
https://asciinema.org/a/bDROwXBDBQQNbTV8Lu7f9jCNs

**Comment by globalchubby** [comment]
OK. So to clarify the expected result is to re-highlight the targets, right?

**Comment by vimeitor** [comment]
Yes, I would expect so. Did you have something else in mind? By the way, thank you very much for this plugin. It works really well and it's very fast.

**Comment by globalchubby** [comment]
No just wanted to confirm.

You're welcome.

Are you interested in taking a shot at a fix?

May not have time to get to this in the next few days.

**Comment by vimeitor** [comment]
I'm submitting a fix in a few minutes. Can you make sure I'm not breaking anything? This seems to fix it.

**Comment by globalchubby** [comment]
Do you have the same bug when you use `(snipe-f)`? I can't seem to reproduce with the mapping

```vim
map <leader><leader>f <Plug>(snipe-f)
```

can you verify?

Here's the result for `(snipe-f)`.

![bug](https://user-images.githubusercontent.com/2729079/33751209-3f22843a-db8e-11e7-800c-c791eab2d306.gif)

`(snipe-t)` is the same.

**Comment by vimeitor** [comment]
Cannot reproduce the issue either with your mapping.
